block/src/Pos/GState/SanityCheck.hs

```diff
@@ -6,7 +6,7 @@ module Pos.GState.SanityCheck
 
 import           Universum
 
-import           Control.Monad.Catch (MonadMask)
+import           Control.Exception.Safe (MonadMask)
```

Comment:
I do not quite get it. `Universum` experts `MonadMask` from `exceptions-safe` already.
https://github.com/serokell/universum/blob/master/src/Universum/Exception.hs#L18